numpy/linalg/linalg.py

```diff
@@ -531,6 +537,40 @@ def inv(a):
            [[-1.25,  0.75],
             [ 0.75, -0.25]]])
 
+    If a matrix is close to singular, the computed inverse may not satisfy
+    ``dot(a, ainv) = dot(ainv, a) = eye(a.shape[0])`` even if a `LinAlgError`
+    is not raised:
+
+    >>> a = np.array([[2,4,6],[2,0,2],[6,8,14]])
+    >>> inv(a)  # No errors raised
+    array([[-1.12589991e+15, -5.62949953e+14,  5.62949953e+14],
+       [-1.12589991e+15, -5.62949953e+14,  5.62949953e+14],
+       [ 1.12589991e+15,  5.62949953e+14, -5.62949953e+14]])
+    >>> np.dot(a, inv(a))
+    array([[ 0.   , -0.5  ,  0.   ],
+           [-0.5  ,  0.625,  0.25 ],
+           [ 0.   ,  0.   ,  1.   ]])
+
+    To detect ill-conditioned matrices, you can use `numpy.linalg.cond` to
+    compute its `condition number`. The larger the condition number, the more
+    ill-conditioned the matrix is. As a rule of thumb, if the condition number
+    `cond(a) = 10**k`, then you may lose up to `k` digits of accuracy on top of
+    what would be lost to the numerical method due to loss of precision from
+    arithmetic methods. See https://en.wikipedia.org/wiki/Condition_number
```

Comment:
It might be a bit nicer to add it in a reference section.  It probably is even an OK reference in the first note (as the article also explains "ill-conditioned").